CosmoTech_Acceleration_Library/Accelerators/scenario_download/scenario_downloader.py

```diff
@@ -67,19 +117,26 @@ def download_dataset(self, dataset_id: str) -> (str, str, Union[str, None]):
             parameters = dataset['connector']['parameters_values']
 
             is_adt = 'AZURE_DIGITAL_TWINS_URL' in parameters
-            is_twin_cache = 'TWIN_CACHE_NAME' in parameters
+            is_twingraph = dataset['twingraph_id'] is not None
+            is_legacy_twin_cache = 'TWIN_CACHE_NAME' in parameters  # Legacy twingraph dataset with specific connector
 
             if is_adt:
                 return {
                     "type": 'adt',
                     "content": self._download_adt_content(
                         adt_adress=parameters['AZURE_DIGITAL_TWINS_URL']),
                     "name": dataset['name']}
-            elif is_twin_cache:
+            elif is_twingraph:
+                return {
+                    "type": "adt",
```

Comment:
Is the `adt` still valid/make sense here? Should it be `redis` (or better `twingraph` since we may not keep redis graph long)? Or is it still the "adt" data format (aka `is_adt`/`is_twingraph`/`is_legacy_twin_cache` produce the same data "adt" format)?

Reply:
I agree this should probably be changed, I just kept the value previously used for both ADT and "legacy" (v2) twingraph :slightly_smiling_face: 
I'm not sure if we internally refer to the output format as an "adt graph format", or if it was just a convenient copy/pasted value. This "adt" value is only used to trigger the call to the `adt_dataset` method, which may need to be renamed as well